analytics_data_api/v0/serializers.py

```diff
@@ -75,6 +75,43 @@ class Meta(object):
         )
 
 
+class ConsolidatedAnswerDistributionSerializer(ProblemResponseAnswerDistributionSerializer):
+    """
+    Serializer for consolidated answer distributions.
+    """
+
+    consolidated_variant = serializers.BooleanField()
+
+    class Meta(ProblemResponseAnswerDistributionSerializer.Meta):
+        fields = (
```

Comment:
I can't access parent attributes in the class block, so I need to redeclare fields here to include consolidated_variants.